src/clients/BundleDataClient/BundleDataClient.ts

```diff
@@ -911,7 +924,7 @@ export class BundleDataClient {
                     v3RelayHashes[relayDataHash].fill = fillToRefund;
```

Comment:
What if I use unsafeDeposit to make a two, duplicate, normal deposits. I then fill one of them. It looks like I will be refunded once here, and then my second deposit is absorbed into the LP fund. Do we need to basically mimic a duplicate fill and push multiple, identical fills into `validatedBundleV3Fills`?

Reply:
hmm good point. We probably should add a check here to refund all duplicate deposits in the bundle once we have seen the fill. I think we can add back this logic here: https://github.com/across-protocol/sdk/pull/835/commits/e5cd4189c7172309b36cf608489ef981f9be4283